src/reducers/filterMarkers.js

```diff
@@ -98,18 +100,40 @@ export default (state = initialState, act) => {
       };
 
     case actions.SET_FILTER_FUNCTION:
-      return { filterFunction: !state.filterFunction, ...state };
+      return { ...state, filterTags: [...state.filterTags, act.tag] }
+
+    case actions.SET_ENTRY_FILTER_FUNCTION:
```

Comment:
This might change to something like `actions.SOMETHING.type`, this file should have some examples

Reply:
fixed